### PR TITLE
MGMT-1764 - move ignition generation to assisted-ignition-generator

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: internal/versions/versions.go [hunk 1]
```diff
@@ -33,12 +33,12 @@ func (h *handler) ListComponentVersions(ctx context.Context, params operations.L
...
-				"assisted-installer-service":                 h.versions.SelfVersion,
-				"image-builder":                              h.versions.ImageBuilder,
-				"discovery-agent":                            h.versions.AgentDockerImg,
-				"ignition-manifests-and-kubeconfig-generate": h.versions.KubeconfigGenerator,
-				"assisted-installer":                         h.versions.InstallerImage,
-				"assisted-installer-controller":              h.versions.ControllerImage,
+				"assisted-installer-service":    h.versions.SelfVersion,
+				"image-builder":                 h.versions.ImageBuilder,
+				"discovery-agent":               h.versions.AgentDockerImg,
+				"assisted-ignition-generator":   h.versions.KubeconfigGenerator,
```
Comment (eranco74) [inline internal/versions/versions.go:39]: Consider renaming h.versions.KubeconfigGenerator, while you are at it

File: pkg/job/job.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const kubeconfigPrefix = "generate-kubeconfig"
+const kubeconfigPrefix = "ignition-generator"
```
Comment (eranco74) [inline pkg/job/job.go:27]: Rename the const as well

File: pkg/job/job.go [hunk 3]
```diff
@@ -57,7 +57,7 @@ type Config struct {
 	JobMemoryLimit      string        `envconfig:"JOB_MEMORY_LIMIT" default:"1000Mi"`
 	JobCPURequests      string        `envconfig:"JOB_CPU_REQUESTS" default:"300m"`
 	JobMemoryRequests   string        `envconfig:"JOB_MEMORY_REQUESTS" default:"400Mi"`
-	KubeconfigGenerator string        `envconfig:"KUBECONFIG_GENERATE_IMAGE" default:"quay.io/ocpmetal/ignition-manifests-and-kubeconfig-generate:stable"` // TODO: update the latest once the repository has git workflow
+	KubeconfigGenerator string        `envconfig:"KUBECONFIG_GENERATE_IMAGE" default:"quay.io/ocpmetal/assisted-ignition-generator:stable"` // TODO: update the latest once the repository has git workflow
```
Comment (eranco74) [inline pkg/job/job.go:60]:
```suggestion
	IgnitionGenerator string        `envconfig:"IGNITION_GENERATE_IMAGE" default:"quay.io/ocpmetal/assisted-ignition-generator:stable"` // TODO: update the latest once the repository has git workflow
```

File: tools/deploy_assisted_installer_configmap.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def main():
 
             versions = {"IMAGE_BUILDER": "installer-image-build",
                         "AGENT_DOCKER_IMAGE": "agent",
-                        "KUBECONFIG_GENERATE_IMAGE": "ignition-manifests-and-kubeconfig-generate",
+                        "KUBECONFIG_GENERATE_IMAGE": "assisted-ignition-generator",
```
Comment (eranco74) [inline tools/deploy_assisted_installer_configmap.py:46]:
```suggestion
                        "IGNITION_GENERATE_IMAGE": "assisted-ignition-generator",
```

Comment (yevgeny-shnaidman) [reply tools/deploy_assisted_installer_configmap.py]: fixed

File: pkg/job/job.go (hunk 3 above)
Comment (yevgeny-shnaidman) [reply pkg/job/job.go]: fixed

File: pkg/job/job.go (hunk 2 above)
Comment (yevgeny-shnaidman) [reply pkg/job/job.go]: fixed

File: internal/versions/versions.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply internal/versions/versions.go]: fixed

**Pull request merged**